src/views/App.js

```diff
@@ -96,14 +97,20 @@ class App extends Component<AppP> {
         headers: this.getQueryHeaders()
       })
     } else {
+      // includeTableSchema must be passed as a body param when executing an UNSAVED query
       const unsavedQueryURL = `${API_HOST}/v0/${store.queryType}/${
         store.dataset
-      }?includeTableSchema=true`
+      }`
+
+      const bodyParams = {
+        query: store.query,
+        includeTableSchema: true
+      }
 
       res = await fetch(unsavedQueryURL, {
         method: 'POST',
         headers: this.getQueryHeaders(),
-        body: createParams({ query: store.query }).toString()
+        body: JSON.stringify(bodyParams)
```

Comment:
TBH I have no experience with this code base, but happy to jump into help. Do you have steps on how I can test this?